### ISSUE TITLE
releasing a stable version

### ISSUE THREAD

**Issue by Raphy** [Taluu/nutwerk-orm-extension#5]
I'm currently using Silex and I want to use Doctrine ORM.

I've found your package and it provides what I want :)

The problem is I always use « stable » packages and your package is not « stable ».
To make him « stable », you have to make a release (« v1.0.0 » maybe ;) )

So it's possible to create a release ?

**Comment by Taluu** [comment]
Hi,

The thing is, this package is not maintained anymore. I can do a tag if I must, but I'd rather recommend you to go for another maintained extension, such as @simensen's extension that can be found here : https://github.com/dflydev/dflydev-doctrine-orm-service-provider

Cheers

**Comment by simensen** [comment]
@Taluu You could consider marking the package deprecated on packagist. That might help people find it in a more automated way. :) I think you can even mark it as "replaced by" so you can send people directly to dflydev/doctrine-orm-service-provider if you want to.

**Comment by Taluu** [comment]
True, I thought I already did that but it seems I didn't. I just marked it as abandonned and marked your extension as the suggested replacement.

Thanks for reminding me to mark it as abandonned. :}
